syncer.go

```diff
@@ -142,7 +142,8 @@ func (s *Syncer) LoadClients() error {
 		if ok {
 			if syncerEntry.ClientConfig == clientConfig {
 				// Exists, and the same config.
-				syncerEntry.Client.RebuildClient()
+				err := syncerEntry.Client.RebuildClient()
+				s.logger.WithError(err).Warnf("Unable to rebuild client")
```

Comment:
Does `WithError` only execute if `err` is present?  Otherwise this will warn on every iteration.